cypress/integration/all/app/queryflow.spec.ts

```diff
@@ -9,140 +9,163 @@ describe(
+      cy.get(
+        ".schedule-editor-modal__form-field--frequency > .dropdown__select"
+      ).click();
 
-      // cy.get(".schedule-editor-modal__form-field--shard > .input-field")
-      //   .click()
-      //   .type("50");
+      cy.findByText(/every 6 hours/i).click();
 
-      // cy.get(".schedule-editor-modal__btn-wrap")
-      //   .contains("button", /schedule/i)
-      //   .click();
+      cy.findByText(/show advanced options/i).click();
 
-      //   cy.visit("/schedule/manage");
+      cy.findByText(/ignore removals/i).click();
+      cy.findByText(/snapshot/i).click();
 
-      //   cy.wait(3000); // eslint-disable-line cypress/no-unnecessary-waiting
-      //   cy.findByText(/query all window crashes/i).should("exist");
+      cy.get(".schedule-editor-modal__form-field--shard > .input-field")
+        .click()
+        .type("{selectall}{backspace}10");
 
-      //   cy.findByText(/actions/i).click();
-      //   cy.findByText(/edit/i).click();
+      cy.get(".schedule-editor-modal__btn-wrap")
+        .contains("button", /schedule/i)
+        .click();
 
-      //   cy.get(
-      //     ".schedule-editor-modal__form-field--frequency > .dropdown__select"
-      //   ).click();
+      // Start e2e tests for team schedules
+      cy.visit("/settings/teams");
 
-      //   cy.findByText(/every 6 hours/i).click();
+      cy.wait(2000); // eslint-disable-line cypress/no-unnecessary-waiting
+      cy.findByRole("button", { name: /create team/i }).click();
 
-      //   cy.findByText(/show advanced options/i).click();
+      cy.findByLabelText(/team name/i)
+        .click()
+        .type("Apples");
 
-      //   cy.findByText(/ignore removals/i).click();
-      //   cy.findByText(/snapshot/i).click();
+      cy.get(".create-team-modal__btn-wrap")
+        .contains("button", /create/i)
+        .click();
 
-      //   cy.get(".schedule-editor-modal__form-field--shard > .input-field")
-      //     .click()
-      //     .type("{selectall}{backspace}10");
+      cy.visit("/schedule/manage");
 
-      //   cy.get(".schedule-editor-modal__btn-wrap")
-      //     .contains("button", /schedule/i)
-      //     .click();
+      cy.wait(2000); // eslint-disable-line cypress/no-unnecessary-waiting
+      cy.findByText(/all teams/i).click();
+      cy.findByText(/apples/i).click();
 
-      //   cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
-      //   cy.findByText(/actions/i).click();
-      //   cy.findByText(/remove/i).click();
+      cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
+      cy.findByText(/query all window crashes/i).should("not.exist");
+      cy.findByText(/inherited query/i).click();
+      cy.findByText(/query all window crashes/i).should("exist");
 
-      //   cy.get(".remove-scheduled-query-modal__btn-wrap")
-      //     .contains("button", /remove/i)
-      //     .click();
+      // End e2e test for team schedules
```

Comment:
This is in the `cypress/integration/all` directory which means the test is run on both Free and Premium tiers. This will need to be moved into a test file in `cypress/integration/premium`, as otherwise it gets an access denied page. I'm not sure why you are getting such an unhelpful error from the CI run, but this can be clearly seen when running `yarn e2e-browser:free` locally.